tests/NHSD.BuyingCatalouge.Ordering.Api.Testing.Data/Entities/AddressEntity.cs

```diff
@@ -38,5 +40,22 @@ public sealed class AddressEntity : EntityBase
                 @Country
             );
             SELECT SCOPE_IDENTITY();";
+
+        public static async Task<AddressEntity> FetchAddressByAddressId(string connectionString, int? addressId)
+        {

```

Comment:
Is there any point executing the database query If the `addressId` is null? Could maybe put a guard in if that's the case i.e. if addressId is null, return null.